src/controllers/mainController.ts

```diff
@@ -151,6 +151,15 @@ export default class MainController implements vscode.Disposable {
 
 			this.initializeObjectExplorer();
 
+			this.registerCommandWithArgs(Constants.cmdConnectObjectExplorerProfile);
+			this._event.on(Constants.cmdConnectObjectExplorerProfile, async (profile: IConnectionProfile) => {
+				await this.createObjectExplorerSession(profile).then(async (result) => {
+					if (result === true) {
+						await this._connectionMgr.connectionUI.saveProfile(profile);
+					}
+				});
```

Comment:
more async/await cleanup
```suggestion
				const result = await this.createObjectExplorerSession(profile);
				if (result === true) {
					await this._connectionMgr.connectionUI.saveProfile(profile);
				}
```